src/path.rs

```diff
@@ -46,6 +66,50 @@ fn convert_value(value: &async_graphql::Value) -> Option<Cow<'_, str>> {
     }
 }
 
+fn convert_urlencoded(value: &async_graphql::Value) -> Option<Cow<'_, str>> {
+    match value {
+        async_graphql::Value::String(s) => Some(serde_urlencoded::to_string(s).unwrap_or_default()),
+        async_graphql::Value::Number(n) => Some(n.to_string()),
+        async_graphql::Value::Boolean(b) => Some(b.to_string()),
+        async_graphql::Value::Object(map) => {
+            Some(serde_urlencoded::to_string(json!(map).to_string()).unwrap_or_default())
+        }
+        async_graphql::Value::List(list) => {
+            Some(serde_urlencoded::to_string(json!(list).to_string()).unwrap_or_default())
+        }
+        _ => None,
+    }
+    .map(Cow::Owned)
```

Comment:
If we use Owned everwhere, there will be no gain in performance. Wisely use Borrow wherever possible.

Reply:
I think it may have been a mistake to include create the `PathUrlEncoded` trait. Graphql values shouldn't need url encoding, right?